### PR TITLE
Support deduplicating based on Postgres LSN

### PR DESCRIPTION
**Pull request by umanwizard**
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/3771)

### CODE REVIEW

File: src/interchange/src/avro.rs [hunk 1]
```diff
@@ -1182,10 +1213,11 @@ impl DebeziumDecodeState {
                         .ok_or_else(|| anyhow!("\"row\" is not an integer"))?;
                         let pos = usize::try_from(pos_val)?;
                         let row = usize::try_from(row_val)?;
+                        // TODO(btv) Add LSN handling here too (OCF code path).
+                        // Better yet, just delete this and make it go through the same code path as Kafka
                         if !self.dedup.should_use_record(
                             &file_val,
-                            pos,
-                            row,
+                            RowCoordinates::MySql { pos, row },
```
Comment (nacrooks) [inline src/interchange/src/avro.rs:1220]: Just to make sure I understand, does this mean that OCF + Postgres won't currently work right now?

**Comment by nacrooks** [comment]
Looks good to me!

File: src/interchange/src/avro.rs (hunk 1 above)
Comment (umanwizard) [reply src/interchange/src/avro.rs:1220]: If tb is outputting a `source` stanza, it won't work -- I need to look into this to see.

Comment (umanwizard) [reply src/interchange/src/avro.rs:1220]: Actually, nevermind. It will work "fine", but it won't perfrorm any deduplication.

**Pull request merged**